spring-cloud-data-rest/src/main/java/org/springframework/cloud/data/rest/AdminApplication.java

```diff
@@ -18,13 +18,14 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.annotation.ComponentScan;
+import org.springframework.cloud.data.rest.config.AdminConfiguration;
+import org.springframework.context.annotation.Import;
 
 /**
  * @author Mark Fisher
  */
 @SpringBootApplication
-@ComponentScan(basePackages = "org.springframework.cloud.data.rest.controller")
```

Comment:
I think for now we should keep it as simple as possible. We can add componentscan back on the AdminConfiguration class if/when needed.